app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/cohort/CohortPixelInterceptor.kt

```diff
@@ -77,6 +77,7 @@ class CohortPixelInterceptor @Inject constructor(
             "m_atp_ev_cpu_usage_above_",
             "m_atp_unprotected_apps_bucket_",
             "m_atp_breakage_report",
+            "m_atp_tds_experiment_download_failed",
```

Comment:
I don't think this is the interceptor that you are looking for, I think you want `VpnPixelParamRemovalPlugin`